workspace/viewer/PdfJsViewer.ts

```diff
@@ -364,6 +362,14 @@ export class PdfJsViewer {
     }
   }
 
+  public showAll(url:string, isResize: boolean = false){
+    var pdfViewer = this;
```

Comment:
In TypeScript you don't need these tricks. Use [fat arrows](https://basarat.gitbooks.io/typescript/docs/arrow-functions.html) to bind callback functions to proper `this`

Reply:
Done.